### ISSUE TITLE
Change the event time

### ISSUE THREAD

**Issue by kristjanmik** [jsis/reykjavikjs-website#5]
Currently we have a weekly meetup starting at 17:30 every Tuesday. I suggest that we make it biweekly and start at 19:30.

What do you think? is it better to host it before or after dinner?

**Comment by jonrh** [comment]
I agree on a biweekly schedule. Starting at somewhere between 18:30 and 20:00 sounds best to me. So after dinner. As to a location, KEX is a really cool place but it's so frequently fully booked or reserved. If we go to a biweekly schedule and move to Facebook events I think it could be a cool idea to swap the places around where we meet, to mix things up a bit. Here are some of my favourite places I think may be suitable: 

* Loft Hostel
* Stofan
* Nora
* American Bar
* Brooklin Bar 

Btw I thought we were meeting on Wednesdays, am I perhaps confusing this event to other?

**Comment by kristjanmik** [comment]
No, my bad. I was confusing this one to another one :P
Should we try to do from 19:30 to 21:30 this Wednesday(8th of July)?
@jonrh isn't loft booked often as well?
